src/tapis-hooks/files/useList.ts

```diff
@@ -0,0 +1,36 @@
+import { useInfiniteQuery } from 'react-query';
+import { list } from 'tapis-api/files';
+import { Files } from '@tapis/tapis-typescript'
+import { useTapisConfig } from 'tapis-hooks';
+import QueryKeys from './queryKeys';
+
+type useListParams = {
+  systemId: string,
+  path: string,
+  limit?: number
+}
+
+const useList = ({ systemId, path, limit = 100 } : useListParams) => {
+  const { accessToken, basePath } = useTapisConfig();
+  const params: Files.ListFilesRequest = {
+    systemId,
+    path,
+    limit
+  };
+  const result = useInfiniteQuery<Files.FileListingResponse, Error>(
+    [QueryKeys.list, params, accessToken],
+    // Default to no token. This will generate a 403 when calling the list function
+    // which is expected behavior for not having a token
+    ({ pageParam = params }) => list(pageParam, basePath, accessToken?.access_token || ''),
```

Comment:
Probably a super nitpick, but could we switch to using the `??` operator to signal that the intention is to catch null/undefined values (as opposed to generically falsy values)?